### PR TITLE
Quiz questions

### PR DESCRIPTION
**Pull request by codewithrishu**
made questions and result page

deploy-link : https://sharp-tesla-021a26.netlify.app/questions/question1.html

### CODE REVIEW

**Review by Hnh013 — APPROVED** [review]
good to merge !

**Review by thegeorgenikhil — APPROVED** [review]
Good to merge! prettify the css files as some extra spacing is there.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+        <div class="quiz-cateogory" id="cateogry">
+            <div class="card">
+                <div class="dismiss-icon">
+                    <span class="material-icons">
+                        favorite_border
+                    </span>
+                </div>
+                <div class="card-image">
+                    <img class="image" src="images/friends.jpg" alt="">
+                </div>
```
Comment (lomesshh) [inline index.html:55]: give proper value to alt attribute

**Review by lomesshh — APPROVED** [review]
everything else looks good to me

File: index.html (hunk 1 above)
Comment (codewithrishu) [reply index.html:55]: done

**Pull request merged**